server/routes/profile.js

```diff
@@ -12,22 +12,24 @@ const {
   savePhoto,
   deletePhoto,
   getProfileFromUserId,
+  searchSitterProfiles,
 } = require("../controllers/profile");
 
 const upload = multer({ dest: "./uploads" });
 
 router.route("/").get(protect, loadProfiles); //get all the profiles
```

Comment:
Suggestion: Did you consider the possibility of leveraging an existing route to query the database?

Since we are searching through the database and returning documents without modifying them, we could use this route with query parameters? :)

Reply:
@BerryGIT-ME You can leave it as is (we can focus on completing other features) but it is a good practice to consider extensibility/reusability when designing routes/controllers.

Also, in the existing route `/search` since we are only searching through the database and returning documents without modifying them, it is a good practice to use query parameters along the HTTP GET method  :)